.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea/*
 *.iml
+.DS_Store
+**/.DS_Store
```

Comment:
What is the reason to have this files at all? why do you need them?

Reply:
Actually, these files are hidden and I don't see them. But I tried to remove them in previous commit.